Resources/Public/JavaScript/Crud.js

```diff
@@ -72,6 +72,8 @@ define(['jquery', 'TYPO3/CMS/FrontendEditing/FrontendEditing'], function ($, Fro
 
 		numberOfRequestsLeft = items.count();
 		items.forEach(function (item) {
+			F.showLoadingScreen();
```

Comment:
This will then be triggered on every Content item. It will be a very expensive call to do this for the DOM structure and Javascript memory consumption. Should still be set in a EVENT like UPDATE_CONTENT_COMPLETE

Reply:
These comments came in just as I was merging the ticket.

showLoadingScreen() doesn't update the DOM more than once, it just increases a counter. I'm adding one for each item so that I can add a progress bar later.